features/nanostack/coap-service/test/coap-service/unittest/stub/mbedtls_stub.c

```diff
@@ -248,6 +248,38 @@ int mbedtls_ctr_drbg_random(void *p_rng,
     return mbedtls_stub.crt_expected_int;
 }
 
+// from hmac_drbg.h
+void mbedtls_hmac_drbg_init( mbedtls_hmac_drbg_context *ctx )
+{
+
+}
```

Comment:
If we use the global RNG, we shouldn't have to add or maintain these stubs, right?